agent/replacecall/src/main/java/randoop/instrument/ReplaceCallAgent.java

```diff
@@ -68,12 +70,30 @@ public class ReplaceCallAgent {
   /** The file from which to read the user replacements for replacing calls. */
   @SuppressWarnings("WeakerAccess")
   @Option("file listing methods whose calls to replace by substitute methods")
-  public static @Nullable Path replacement_file = null;
+  public static @MonotonicNonNull Path replacement_file = null;
 
   /** Exclude transformation of classes in the listed packages. */
   @SuppressWarnings("WeakerAccess")
   @Option("file listing packages whose classes should not be transformed")
-  public static @Nullable Path dont_transform = null;
+  public static @MonotonicNonNull Path dont_transform = null;
+
+  /**
+   * Default replacement methods that should not be used. A comma-separated list of method
+   * signatures in colon/JVM-descriptor form, e.g. "java.lang.System.exit:(I)V". JVM method
```

Comment:
Where is "colon/JVM-descriptor form" documented?  Please provide a URL or other reference.  (For example, method descriptors are defined in JVMS section 4.3.3, and Randoop fully-qualified signatures are defined at https://randoop.github.io/randoop/manual/#fully-qualified-signature.)  If you made it up, then I suggest using a standard format instead.

It is odd that one signature format is used for adding a method replacement (namely, Randoop fully-qualified signatures, defined at https://randoop.github.io/randoop/manual/#fully-qualified-signature), but a *different* signature format is used for excluding one.  I think that difference is likely to confuse users.  What is the benefit of using a different format?